sdk/servicebus/azure-servicebus/azure/servicebus/aio/management/_management_client_async.py

```diff
@@ -524,7 +523,6 @@ def entry_to_qd(entry):
         get_next = functools.partial(
             get_next_template,
             functools.partial(self._impl.list_entities, constants.ENTITY_TYPE_QUEUES),
-            api_version=self._api_version,
```

Comment:
passing in api version is no longer required ?

Reply:
it's passed into the client, then passed internally